contracts/Safe.sol

```diff
@@ -292,7 +292,7 @@ contract Safe is
         // Load threshold to avoid multiple storage loads
         uint256 _threshold = threshold;
         // Check that a threshold is set
-        require(_threshold > 0, "GS001");
+        if (_threshold == 0) revertWithError("GS001");
```

Comment:
This is such a surprise to me that changing `require` to `if` affects the code size...

Reply:
I see now, its because of how the `require` was encoding the revert error.